source/leuven/tier2_hardware/wice_hardware.rst

```diff
@@ -44,3 +44,41 @@ The nodes are connected using an Infiniband HDR-100 network, the islands are ind
 
 .. figure:: wice_hardware/wice.png
```

Comment:
Perhaps the diagram should be extended as well, maybe there is already an Excel showing the physical location of the new nodes.

Reply:
Yes we should do that eventually, but I propose to do that when going into production (as mentioned at the top)